src/commands/database-rules-get.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from "../command";
+import * as logger from "../logger";
+import * as requireInstance from "../requireInstance";
+import { requirePermissions } from "../requirePermissions";
+import * as metadata from "../database/metadata";
+
+export default new Command("database:rules:get <id>")
```

Comment:
This is the only parameter that is just `id` - is that intentional?

Reply:
nope, changed to `rulesetId`